docs/send-data/kubernetes/logs.md

```diff
@@ -0,0 +1,553 @@
+```yaml
+sumologic:
+  logs:
+    container|systemd|kubelet:
+      otelcol:
+        extraProcessors:
+          - transform/mask-card-numbers:
+              log_statements:
+                - context: log
+                  statements:
+                    - replace_pattern(body, "card=\\d+", "card=***")
+```
+
+To modify record attributes, use the [Attributes processor][attributes_processor_docs]:
+
+```yaml
+sumologic:
+  logs:
+    container|systemd|kubelet:
+      otelcol:
+        extraProcessors:
+          - attributes/delete-record-attribute:
+              actions:
+                - action: delete
+                  key: unwanted.attribute
+          # To rename old.attribute to new.attribute, first create new.attribute and then delete old.attribute.
+          - attributes/rename-old-to-new:
+              - action: insert
+                key: new.attribute
+                from_attribute: old.attribute
+              - action: delete
+                key: old.attribute
+```
+
+To modify resource attributes, use the [Resource processor][resource_processor_docs]:
+
+```yaml
+sumologic:
+  logs:
+    container|systemd|kubelet:
+      otelcol:
+        extraProcessors:
+          - resource/add-resource-attribute:
+              attributes:
+                - action: insert
+                  key: environment
+                  value: staging
+          - resource/remove:
+              attributes:
+                - action: delete
+                  key: redundant-attribute
+```
+
+#### Adding custom fields
+
+To add a custom [field][sumo_fields] named `static-field` with value `hardcoded-value` to logs, use the following configuration:
+
+```yaml
+sumologic:
+  logs:
+    container|systemd|kubelet:
+      otelcol:
+        extraProcessors:
+          - resource/add-static-field:
+              attributes:
+                - action: insert
+                  key: static-field
+                  value: hardcoded-value
+```
+
+To add a custom field named `k8s_app` with a value that comes from e.g. the pod label `app.kubernetes.io/name`, use the following
+configuration:
+
+```yaml
+sumologic:
+  logs:
+    container|systemd|kubelet:
+      otelcol:
+        extraProcessors:
+          - resource/add-k8s_app-field:
+              attributes:
+                - action: insert
+                  key: k8s_app
+                  from_attribute: pod_labels_app.kubernetes.io/name
+```
+
+:::note
+Make sure the field is [added in Sumo Logic][sumo_add_fields].
+:::
+
+## Persistence
+
+By default, the metadata enrichment service provisions and uses a Kubernetes PersistentVolume as an on-disk queue that guarantees durability
+across Pod restarts and buffering in case of exporting problems.
+
+This feature is enabled by default, but it only works if you have a correctly configured default `storageClass` in your cluster. Cloud
+providers will do this for you when provisioning the cluster. The only alternative is disabling persistence altogether.
```

Comment:
```suggestion
By default, the metadata enrichment service provisions and uses a Kubernetes PersistentVolume as an on-disk queue that guarantees durability across Pod restarts and buffering in case of exporting problems.

This feature is enabled by default, but it only works if you have a correctly configured default `storageClass` in your cluster. Cloud providers will do this for you when provisioning the cluster. The only alternative is disabling persistence altogether.
```